src/pages/index.js

```diff
@@ -28,10 +28,10 @@ const IndexPage = ({ data: { allEvent } }) => {
       />
       <Panel>
         <LargeParagraph>
-          QueerJS is a series of meetups where anyone is welcome to attend and support the speakers
-          and the idea but all the speakers will be Queer.
-          <br />
-          This meetup exists to give a voice to everyone, to make a safe space where everyone is
+          QueerJS is a meetup series where everyone is welcome to attend and support the speakers
```

Comment:
Reading it again, can we bring back `encouraged` here to replace welcome? Only because we say `everyone is welcome` again in the next sentence. But also not a necessary fix because it's not the worst phrase to have twice 😆